### PR TITLE
Static dispatch output correction

### PR DESCRIPTION
**Pull request by raminfarajpour**

### CODE REVIEW

**Review by mokarchi — APPROVED** [review]

File: backend/dotnet/questions/oop.md [hunk 1]
```diff
@@ -203,8 +203,8 @@
     ```
     Output:
     ```
-    It is an IBar.
-    It is a Bar.
+    It is an Bar.
```
Comment (Hossein13M) [inline backend/dotnet/questions/oop.md]: Consider changing this sentence to `It is a Bar`

**Review by Hossein13M — CHANGES_REQUESTED** [review]
There is a minor grammatical issue in this PR.

**Review by Hossein13M — APPROVED** [review]
thanks @raminfarajpour

**Comment by raminfarajpour** [comment]
Thanks for mentioning @hossein13m 
I correct that grammatical problems and some other problems in dynamic dispatch output

**Review by Hossein13M — APPROVED** [review]

**Review by MoienTajik — APPROVED** [review]

**Pull request merged**